README.md

```diff
@@ -102,17 +102,17 @@ Detailed description of the inputs exposed by the
     # Required
     tool:
 
-    # Token for authenticating requests to SonarCloud.
+    # Token for authenticating requests to Sonar.
     # Required, when tool is "sonar" and "file" has not been set. Only required for private repository.
     sonar-token:
 
-    # Key identifying the SonarCloud component to be analyzed. Only necessary if deviating from SonarCloud's established convention.
+    # Key identifying the Sonar component to be analyzed. Only necessary if deviating from Sonar's established convention.
     # Default: `owner_repo`
     sonar-component-key:
 
-    # Base URL of the Sonar API. Use this to switch from SonarCloud to SonarQube.
-    # Default: https://sonarcloud.io/api
-    sonar-api-url:
+    # Base host of the Sonar or SonarQube API. Use this to switch from Sonar to SonarQube.
```

Comment:
"Base host" is confusing. In this context, let's avoid the ambiguous "Sonar": we mean to differentiate between SonarCloud and SonarQube.

```suggestion
    # SonarCloud or SonarQube host URL. Use this to switch from SonarCloud to SonarQube.
```